src/Areas/LoadTesting/Services/LoadTestingService.cs

```diff
@@ -0,0 +1,287 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Text.Json.Serialization;
+using Azure;
+using Azure.Core;
+using Azure.Developer.LoadTesting;
+using Azure.ResourceManager;
+using Azure.ResourceManager.LoadTesting;
+using Azure.ResourceManager.Resources;
+using AzureMcp.Areas.LoadTesting.Commands;
+using AzureMcp.Areas.LoadTesting.Models.LoadTest;
+using AzureMcp.Areas.LoadTesting.Models.LoadTestingResource;
+using AzureMcp.Areas.LoadTesting.Models.LoadTestRun;
+using AzureMcp.Options;
+using AzureMcp.Services.Azure;
+
+namespace AzureMcp.Areas.LoadTesting.Services;
+
+public class LoadTestingService() : BaseAzureService, ILoadTestingService
+{
+
+    /* Load Testing Resource APIs Start */
+    public async Task<List<TestResource>> GetLoadTestResourcesAsync(string subscriptionId, string? resourceGroup = null, string? testResourceName = null, string? tenant = null, RetryPolicyOptions? retryPolicy = null)
+    {
+        ValidateRequiredParameters(subscriptionId);
```

Comment:
You always need to resolve subscription parameter using the GetSubscription method because it could be id or name.